boards/PDM/Inc/App/App_CurrentLimits.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#define AUX1_MIN_CURRENT 0.0f
```

Comment:
I needed an elegant way to share these constants between embedded and gtest targets so i didn't have to hard code the same value twice. This was the simplest way I could think of.